remotemonitoring/armtemplates/local.json

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
Overall question about local vs. static map.

Do we need to make a distinction between the two for local deployments? We might be able to modify the cli logic so that only one template is needed.

Reply:
I agree with @jillcary for local development i think we should just have one template call it local but without maps. So same as static map

Reply:
I think you can add the logic here https://github.com/Azure/pcs-cli/pull/244/files#diff-a25cfaebe1be93b318c70debfe07a223R95 to check if you should be using local.json or not

Reply:
good point. Updated to keep only one template for local deployments.